cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -58,14 +58,17 @@ type NodeGroupAutoscalingOptions struct {
 
 // GCEOptions contain autoscaling options specific to GCE cloud provider.
 type GCEOptions struct {
-	// ConcurrentRefreshes is the maximum number of concurrently refreshed instance groups or instance templates.
+	// ConcurrentRefreshes is the maximum number of concurrently refreshed instance groups or instance templates or zones with mig instances
 	ConcurrentRefreshes int
 	// MigInstancesMinRefreshWaitTime is the minimum time which needs to pass before GCE MIG instances from a given MIG can be refreshed.
 	MigInstancesMinRefreshWaitTime time.Duration
 	// DomainUrl is the GCE url used to make calls to GCE API.
 	DomainUrl string
 	// LocalSSDDiskSizeProvider provides local ssd disk size based on machine type
 	LocalSSDDiskSizeProvider gce_localssdsize.LocalSSDSizeProvider
+	// BulkMigInstancesListingEnabled means that mig instances should be listed using a single api call and sort them into migs instead of using mig specific api call.
```

Comment:
```suggestion
	// BulkMigInstancesListingEnabled means that cluster instances should be listed in bulk instead of per mig.
```
I'd avoid saying that it's a single API call since it's more complicated than that (per zone, paging etc).

Reply:
Updated